src/components/NavBar/LightDarkSlider.tsx

```diff
@@ -0,0 +1,60 @@
+import { FormLabel, Switch } from "@mui/material";
+import styled from "styled-components";
+import React, { useContext } from "react";
+
+import Sun from "@/components/NavBar/Sun.svg";
+import Moon from "@/components/NavBar/Moon.svg";
+import { ThemeUpdateContext } from "@/app/themes";
+
+const StyledSwitch = styled(Switch)`
```

Comment:
I think if this cannot be refactored any further then it could do with something similar to the table styling in VFB-6 where some comments might need to be made to make the code more understandable. But you probably wanna consult @TomCusack-Softwire for suggestions.

Reply:
Makes sense, will add some comments explaining what different bits are.